lib/core/src/Cardano/Wallet/DB/Store/CBOR/Store.hs

```diff
@@ -0,0 +1,81 @@
+
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE ViewPatterns #-}
+
+{- |
+Copyright: 2022 IOHK
+License: Apache-2.0
+
+Implementation of a 'Store' for 'TxCBORHistory'.
+
+-}
+
+module Cardano.Wallet.DB.Store.CBOR.Store ( mkStoreCBOR ) where
+
+import Prelude
+
+import Cardano.Wallet.DB.Sqlite.Types
+    ( TxId (..) )
+import Cardano.Wallet.DB.Store.CBOR.Model
+    ( DeltaTxCBOR (..), TxCBORHistory (..) )
+import Cardano.Wallet.Primitive.Types.Tx.CBOR
+    ( TxCBOR (..) )
+import Data.ByteString.Lazy.Char8
+    ( fromStrict, toStrict )
+import Data.DBVar
+    ( Store (..) )
+import Data.Generics.Internal.VL
+    ( Iso', fromIso, iso, view )
+import Data.Maybe
+    ( fromJust )
+import Database.Persist
+    ( PersistEntity (keyFromRecordM)
+    , PersistQueryWrite (deleteWhere)
+    , PersistStoreWrite (repsertMany)
+    , entityVal
+    , selectList
+    , (==.)
+    )
+import Database.Persist.Sql
+    ( SqlPersistT )
+
+import qualified Cardano.Wallet.DB.Sqlite.Schema as Schema
+    ( CBOR (..), EntityField (..) )
+import qualified Data.Map.Strict as Map
+
+txCBORiso :: Iso' (TxId, TxCBOR) Schema.CBOR
+txCBORiso = iso f g
+  where
+    f :: (TxId, TxCBOR) -> Schema.CBOR
+    f (id',TxCBOR {..}) =
```

Comment:
```suggestion
    f (id',TxCBOR{..}) =
```

Style: I like to put curly braces `{}` right next to a constructor, so that I can visually see that it's a single argument rather than two.